marketplace/oauthenticator.py

```diff
@@ -0,0 +1,45 @@
+from typing import Tuple
```

Comment:
Would it be possible to maintain this module on the MarketPlace GitLab and install it directly from there? Maybe not for this PR, but in the future?

Reply:
Yes, but only when it reused by others. We can keep an eye on whether Frank going to use this `MarketplaceOauthenticator` again for his jupyterhub deployment.